libs/Microsoft.MixedReality.WebRTC/PeerConnection.cs

```diff
@@ -981,6 +981,36 @@ public void RemoveLocalAudioTrack()
             PeerConnectionInterop.PeerConnection_RemoveLocalAudioTrack(_nativePeerhandle);
         }
 
+        class AudioReadStream : IAudioReadStream
+        {
+            IntPtr _nativeStreamHandle;
+            internal AudioReadStream(IntPtr nativePeerHandle, int bufferMs)
+            {
+                AudioReadStreamInterop.Create(nativePeerHandle, bufferMs, ref _nativeStreamHandle);
+            }
+
+            public void ReadAudio(int sampleRate, float[] data, int channels)
+            {
+                AudioReadStreamInterop.Read(_nativeStreamHandle, sampleRate, data, data.Length, channels);
+            }
+
+            public void Dispose()
+            {
+                AudioReadStreamInterop.Destroy(_nativeStreamHandle);
```

Comment:
Add `GC.SuppressFinalize(this);` and add some finalizer (dtor) calling `Destroy()` too, to avoid any leak? I'm still unclear in C# if this is better practice or if the user not calling `Dispose()` is something that should not be handled.